app/controllers/spree/admin/subscriptions_controller.rb

```diff
@@ -67,8 +67,10 @@ def gather_stats
         this_month_range = Time.zone.now.beginning_of_month..Time.zone.now.end_of_month
         today_range = Time.zone.now.beginning_of_day..Time.zone.now.end_of_day
         tomorrow_range = Time.zone.tomorrow.beginning_of_day..Time.zone.tomorrow.end_of_day
+        active_subs = model_class.where.not(state: ['canceled', 'inactive'])
```

Comment:
not a change request, more of a question.  is there anything in the code base that gives info on why they decided to go with `where.not(state: ['canceled', 'inactive'])` instead of `where(state: 'active')` ?

Reply:
I believe it's because there are multiple states in which subs can be processed. I know there is the `pending_cancellation` state and possibly more where subs are still actionable.

Reply:
right, i saw the `pending_cancellation` state in there, seems like i remember seeing a `paused` state somewhere in there but can't remember.

Reply:
There could be a better way to do it. I was going off of what was in the `actionable` scope on subscription.